analyzers/its/expected/AnalyzeGeneratedVb/AnalyzeGeneratedVb-{C4F122A1-A51D-4194-998F-686A6EF31E44}-S6145.json

```
@@ -0,0 +1,22 @@
+{
+"issues":  [
+{
+"id":  "S6145",
+"message":  "Configure 'Option Strict On' for assembly 'AnalyzeGeneratedVb'.",
+"location":  null
+},
+{
+"id":  "S6145",
+"message":  "Change this to 'Option Strict On'.",
+"location":  {
+"uri": "replaced",
```

Fix rebase conflict (related to change in #4016)
```suggestion
"uri":  "Replaced-Temporary-Path\.NETFramework,Version=v4.5.2.AssemblyAttributes.vb",
```